tests/e2e-kind.sh

```diff
@@ -78,12 +78,12 @@ install_charts() {
 }
 
 pull_and_cache_docker_images() {
-    if ! [ -x "$(command -v yq)" ]; then
```

Comment:
It seems that this check is now removed. It could be useful when testing locally.

Reply:
Was something not working in this logic?

Reply:
oh sorry, now I got the context "Error: Parsing expression: Lexer error: could not match text starting at 1:53 failing at 1:55. unmatched text: "to"" comment.

Reply:
I misunderstood the initial intent of the conditional. Is this change okay?

Reply:
I will merge this to unblock a PR. We can revisit this later if we want to improve the design.

Reply:
Yes that is fine.